### ISSUE TITLE
Logging is missing file name

### ISSUE THREAD

**Issue by necolas** [postcss/postcss-custom-media#5]
This is what I get with the latest version, whereas before you would be told what file the error was in:

```
<css input>:5:1: missing @custom-media definition for '--lg-viewport'. The entire rule has been removed from the output.
```

**Comment by MoOx** [comment]
Fixed and pushed on git, but I am getting issue to publish 2.0.0
https://github.com/npm/npm/issues/7235
I hope to fix that asap.
Meanwhile you can use git tag.

**Comment by MoOx** [comment]
Ref https://github.com/postcss/postcss-custom-media/issues/6

**Comment by MoOx** [comment]
I made some shit when I create the package so it has been published has 3.0.0
